src/contracts/atlas/DAppIntegration.sol

```diff
@@ -57,7 +57,7 @@ contract DAppIntegration {
 
         if (msg.sender != govAddress) revert FastLaneErrorsEvents.OnlyGovernance();
 
-        bytes32 signatoryKey = keccak256(abi.encode(msg.sender, msg.sender));
+        bytes32 signatoryKey = keccak256(abi.encode(msg.sender, controller));
```

Comment:
@thogard785 - Changed `msg.sender` to `controller`. The former version was making `initializeGovernance` revert when the same governance was trying to initialize a new controller.